frontend/package.json

```diff
@@ -16,9 +16,9 @@
 		"clsx": "^2.1.1",
 		"embla-carousel-react": "^8.5.1",
 		"lucide-react": "^0.454.0",
-		"next": "14.2.16",
```

Comment:
Please revert the changes to the package versions in package.json (keep: `next: 14.2.16`, `react: ^18`, `react-dom: ^18`). Do not modify this file. After reverting, delete the `node_modules` folder and `run npm install` to update `package-lock.json`.

Reply:
Done